lib/ansible/plugins/action/yum.py

```diff
@@ -47,7 +48,15 @@ def run(self, tmp=None, task_vars=None):
         del tmp  # tmp no longer has any effect
 
         # Carry-over concept from the package action plugin
-        module = self._task.args.get('use_backend', "auto")
+        module = 'auto'
+
+        if 'use' in self._task.args and 'use_backend' in self._task.args:
```

Comment:
Can you please add a test for this change? Thanks.

Reply:
@Akasurde sure. Can you please point me to any sample test script?

Reply:
You can add test here - https://github.com/ansible/ansible/blob/devel/test/integration/targets/package/tasks/main.yml or https://github.com/ansible/ansible/blob/devel/test/integration/targets/yum/tasks/main.yml

Reply:
done